domain-tier/src/main/java/com/outdoor/foodcalc/domain/service/product/ProductDomainService.java

```diff
@@ -54,15 +54,11 @@ public Optional<Product> getProduct(long id) {
      */
     public Product addProduct(Product product) {
         long id = productRepo.addProduct(product);
-        return new Product(id,
-                product.getName(),
-                product.getDescription(),
-                product.getCategory(),
-                product.getCalorific(),
-                product.getProteins(),
-                product.getFats(),
-                product.getCarbs(),
-                product.getDefaultWeightInt());
+        return  Product.builder().productId(id).name(product.getName())
+                .description(product.getDescription())
+                .category(product.getCategory()).calorific(product.getCalorific())
```

Comment:
please put each statement in a new line
for example change `.category(product.getCategory()).calorific(product.getCalorific())` to
   .category(product.getCategory())
   .calorific(product.getCalorific())